it/uk/gov/hmrc/customsservicestatus/repositories/CustomsServiceStatusRepositoryISpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customsservicestatus.repositories
+
+import uk.gov.hmrc.customsservicestatus.controllers.test.{TestController, routes => testRoutes}
+import uk.gov.hmrc.customsservicestatus.helpers.BaseISpec
+
+class CustomsServiceStatusRepositoryISpec extends BaseISpec {
+
+  val testController: TestController = app.injector.instanceOf[TestController]
+
+  override def beforeEach(): Unit = {
+    super.beforeEach()
+    await(testController.clearAllData(fakeRequest(testRoutes.TestController.clearAllData)))
+  }
+
+  val customsServiceStatusRepository: CustomsServiceStatusRepository = app.injector.instanceOf[CustomsServiceStatusRepository]
+
+  "check" should {
```

Comment:
method no longer called "check" so Spec needs to change.

Reply:
changed.